src/components/modebar/manage.js

```diff
@@ -110,6 +110,7 @@ function getButtonGroups(gd) {
     var hasGL2D = fullLayout._has('gl2d');
     var hasTernary = fullLayout._has('ternary');
     var hasMapbox = fullLayout._has('mapbox');
+    var hasMap = fullLayout._has('map');
```

Comment:
@archmoj I noticed that in some places you have `hasMap = layoutOut._has('mapbox') || layoutOut._has('map');` and in others `hasMap = fullLayout._has('map');` -- is there a reason? Any easy way to make these consistent?

Reply:
Good call. Addressed in [5fc241b](https://github.com/plotly/plotly.js/pull/7060/commits/5fc241bf09eb3200588737963c39e0a2e74787cc).